estate/models/estate_property_tag.py

```diff
@@ -0,0 +1,12 @@
+from odoo import fields, models
+
+
+class PropertyTags(models.Model):
```

Comment:
```suggestion
class PropertyTag(models.Model):
```
Model class names should be singular